additional-providers/hybridauth-authentiq/Providers/Authentiq.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/*!
+* Hybridauth
+* https://hybridauth.github.io | https://github.com/hybridauth/hybridauth
+*  (c) 2017 Hybridauth authors | https://hybridauth.github.io/license.html
+*/
+
+/**
+ * Hybrid_Providers_Authentiq
+ */
+class Hybrid_Providers_Authentiq extends Hybrid_Provider_Model_OAuth2
+{
+  /**
+   * {@inheritdoc}
+   */
+  public $scope = 'openid aq:name email~rs phone~s address aq:push';
+
+  /**
+  * IDp wrappers initializer
+  */
+  function initialize()
+  {
+    parent::initialize();
+
+    // Provider api end-points
+    $this->api->api_base_url  = 'https://connect.authentiq.io/';
+    $this->api->authorize_url = 'https://connect.authentiq.io/authorize';
+    $this->api->token_url     = 'https://connect.authentiq.io/token';
+
+    if( $this->token( 'access_token' ) )
+    {
+      $this->api->curl_header = array( 'Authorization: Bearer ' . $this->token( 'access_token' ) );
+    }
+  }
+
+  /**
+   * begin login step
+   */
+  function loginBegin()
+  {
+    // redirect the user to the provider authentication url
+    Hybrid_Auth::redirect( $this->api->authorizeUrl( array( "scope" => $this->scope, "state" => md5( mt_rand() ) ) ) );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  function logout() {
+      parent::logout();
+  }
+
+  /**
+   * load the user profile from the IDp api client
+   */
+  function getUserProfile()
+  {
+    $data = $this->api->api('userinfo');
+
+    if (!isset( $data->sub )) {
+        throw new Exception('Provider API returned an unexpected response.');
+    }
+
+    $this->user->profile->identifier  = @ (property_exists($data, 'sub')) ? $data->sub : '';
+
+    $this->user->profile->displayName = @ (property_exists($data, 'name')) ? $data->name : '';
+    $this->user->profile->firstName   = @ (property_exists($data, 'given_name')) ? $data->given_name : '';
+    // $this->user->profile->middleName  = @ (property_exists($data, 'middle_name')) ? $data->middle_name : ''; // not supported
+    $this->user->profile->lastName    = @ (property_exists($data, 'family_name')) ? $data->family_name : '';
+
+    if (!$this->user->profile->displayName && $this->user->profile->firstName) {
+        $this->user->profile->displayName = join(' ',
+                                                array(
+                                                  $this->user->profile->firstName,
+                                                  $this->user->profile->lastName
+                                                ));
+    }
+
+    $this->user->profile->email       = @ (property_exists($data, 'email')) ? $data->email : '';
+    $this->user->profile->emailVerified = @ (property_exists($data, 'email_verified')) ? $data->email_verified : '';
```

Comment:
as I see in [response](https://developers.authentiq.com/api-provider/#!/Authentication/userInfo) `email_verified` is boolean, but in HA User object it means `Verified user email` which is string. Here is a [proof](https://github.com/hybridauth/hybridauth/blob/v2/hybridauth/Hybrid/User_Profile.php#L111). 
So, pls use ` $this->user->profile->emailVerified =  $this->user->profile->email;`